lib/lookup.json

```diff
@@ -24,6 +24,33 @@
     "prefix": "v",
     "flaky": true
   },
+  "@yarnpkg/cli": {
+    "envVar": { "TEST_IGNORE_TIMEOUT_FAILURES": true },
+    "install": ["install", "--mode", "skip-build"],
+    "maintainers": ["arcanis", "merceyz", "paul-soporan", "larixer"],
+    "prefix": "@yarnpkg/cli@",
+    "repo": "https://github.com/yarnpkg/berry",
+    "scripts": [
+      "build:cli",
+      "typecheck:all",
+      "test:lint",
+      [
+        "test:unit",
+        "--testPathIgnorePatterns",
+        "./packages/plugin-patch/tests/diffFolders.test.ts",
+        "./packages/acceptance-tests/pkg-tests-specs/sources/commands/patchCommit.test.ts",
+        "./packages/acceptance-tests/pkg-tests-specs/sources/commands/patch.test.ts",
+        "./packages/acceptance-tests/pkg-tests-specs/sources/protocols/git.test.js",
+        "./packages/acceptance-tests/pkg-tests-specs/sources/https.test.js",
+        "./packages/acceptance-tests/pkg-tests-specs/sources/script.test.js"
+      ],
+      "test:integration"
+    ],
+    "flaky": "ppc",
+    "skip": ["aix", "s390x"],
```

Comment:
```suggestion
```
We've recently landed support for big-endian platforms so we can try removing these now.